index.html

```diff
@@ -9064,8 +9050,454 @@
+									</div>
+									<div class="ui-settings-inline custom-mode-font">
+										<div style="margin-right:58px; text-align: center;">You: </div>
+										<div class="enhancedTextColorPicker" id="you-text-colorselector">text🖌️</div>
+										<div class="enhancedTextColorPicker instruct-markdown-user" id="you-speech-colorselector">"speech"🖌️</div>
+										<div class="enhancedTextColorPicker instruct-markdown-user" id="you-action-colorselector">*action*🖌️</div>
+									</div>
+									<div class="ui-settings-inline custom-mode-font">
+										<div style="margin-right:67px; text-align: center;">AI: </div>
+										<div class="enhancedTextColorPicker" id="AI-text-colorselector">text🖌️</div>
+										<div class="enhancedTextColorPicker instruct-markdown-user" id="AI-speech-colorselector">"speech"🖌️</div>
+										<div class="enhancedTextColorPicker instruct-markdown-user" id="AI-action-colorselector">*action*🖌️</div>
+									</div>
+									<div class="ui-settings-inline custom-mode-font">
+										<div style="margin-right:38px; text-align: center;">System: </div>
+										<div class="enhancedTextColorPicker" id="sys-text-colorselector">text🖌️</div>
+										<div class="enhancedTextColorPicker instruct-markdown-user" id="sys-speech-colorselector">"speech"🖌️</div>
+										<div class="enhancedTextColorPicker instruct-markdown-user" id="sys-action-colorselector">*action*🖌️</div>
+									</div>
+									<div class="ui-settings-inline instruct-markdown-user">
+										<div style="margin-right:11px; text-align: center;">Code blocks: </div>
+										<div class="enhancedTextColorPicker" id="code-block-background-colorselector">background🖌️</div>
+										<div class="enhancedTextColorPicker" id="code-block-foreground-colorselector">foreground🖌️</div>
+									</div>
+								</div>
+							</div>
+						</div>
+						<div class="popupfooter" style="margin:-10px; width:110%; margin-top: 200px;">
+							<button type="button" class="btn btn-primary" id="btn_settingsaccept" onclick="hideAestheticUISettingsMenu()">OK</button>
+							<button type="button" class="btn btn-primary" id="btn_settingsclose" onclick="hideAestheticUISettingsMenu()">Cancel</button>
+						</div>
+					</div>
+				</div>
+				<div style="background-color: black; flex: 1; padding: 10px;  padding-bottom: 0px; display: flex; flex-direction: column; max-width: 1000px;">
+					<p>Style Preview</p>
+					<div id="aesthetic_text_preview" style="flex: 0; width: 100%; background-color: black; margin: 2px; text-align: left; word-wrap: break-word;"></div>
+				</div>
+				<input type="color" id="colorPicker" style="display: none;">
+				<input type="color" id="colorPicker_background" style="display: none;">
+				<input type="file" id="portraitFileInput" style="display:none" accept="image/*">
+			</div>
+		</div>
+	</div>
 </body>
+
+<style>
```

Comment:
this really should be defined together with or right below existing styles, same for the script. also I do think it could be a little less messy.